tsconfig.build-esm.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.build.json",
+  "compilerOptions": {
+    "lib": [
+      "es6"
```

Comment:
FWIW I don't think we need to go as far down as ES5 or ES6 - our minimum support is Node 12 so it can be ES2019.

Reply:
Thanks, I'll give it a try